src/main/java/uk/gov/gchq/palisade/resource/request/GetResourcesByResourceRequest.java

```diff
@@ -41,45 +42,49 @@ public GetResourcesByResourceRequest() {
      * @return the {@link GetResourcesByResourceRequest}
      */
     public GetResourcesByResourceRequest resource(final Resource resource) {
```

Comment:
This needs the @generated annotation on the mutator

Reply:
Done